data/en/test_routing_answer_count.json

```diff
@@ -0,0 +1,167 @@
+                    "type": "Question",
+                    "id": "household-composition",
+                    "questions": [{
+                        "id": "household-composition-question",
+                        "title": "Who usually lives here?",
+                        "description": "<br> <div> <h3>Include:</h3> <ul> <li>Yourself, if this is your permanent or family home </li> <li>Family members including partners, children and babies born on or before 9 April 2017</li> <li>Students and/or school children who live away from home during term time</li> <li>Housemates, tenants or lodgers</li> </ul> </div>",
+                        "type": "RepeatingAnswer",
+                        "answers": [{
+                                "id": "first-name",
+                                "label": "First Name",
+                                "mandatory": false,
+                                "q_code": "1",
+                                "type": "TextField"
+                            },
+                            {
+                                "id": "last-name",
+                                "label": "Last Name",
+                                "mandatory": false,
+                                "q_code": "1",
+                                "type": "TextField"
+                            }
+                        ]
+                    }],
+                    "title": "Household",
+                    "routing_rules": [{
+                            "goto": {
+                                "group": "group1",
+                                "when": [{
+                                    "answer_count": "first-name",
+                                    "condition": "equals",
+                                    "value": 2
+                                }]
+                            }
+                        },
+                        {
+                            "goto": {
+                                "group": "group2",
+                                "when": [{
+                                    "answer_count": "first-name",
+                                    "condition": "greater than",
+                                    "value": 2
+                                }]
+                            }
+                        },
+                        {
+                            "goto": {
+                                "group": "group0"
+                            }
+                        }
+                    ]
+                }],
+                "id": "multiple-questions-group",
+                "title": "Routing control group"
+            },
+            {
+                "id": "group0",
+                "title": "This is Group 0 - You answered less than \"2\"",
+                "blocks": [{
+                    "type": "Question",
+                    "id": "group0-block",
+                    "description": "",
+                    "title": "Did you want Group 0?",
```

Comment:
this should probably have ` - You answered less than \"2\"",` on it for consistency and easier testing